pages/search.js

```diff
@@ -128,9 +162,9 @@ export default function Search({ data }) {
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Search</h1>
 
-        <div className="flex flex-wrap justify-center space-x-3 mb-4">
-          {tags &&
-            tags
+        <article className="flex flex-wrap justify-center mb-4">
```

Comment:
Sorry to be a bit picky here, but I'm trying to understand the change to an article html component from a div.  @EmmaDawsonDev can you weigh in on if this makes sense semantically? I'm thinking no becuase it's a group of tags. I would think leaving div would have been appropriate here but not sure for something like this.

Reply:
https://github.com/EddieHubCommunity/LinkFree/pull/5534#pullrequestreview-1368018671

Reply:
> Sorry to be a bit picky here, but I'm trying to understand the change to an article html component from a div.  @EmmaDawsonDev can you weigh in on if this makes sense semantically? I'm thinking no becuase it's a group of tags. I would think leaving div would have been appropriate here but not sure for something like this.

Sorry for the late reply! Article should be used for something self contained when something can be completely lifted out from where it is, moved to another place and still make sense like a blog article. I don't think it makes sense in this case and a div is better.

Reply:
> #5534 (review)

@joe-shajan I agree this should remain as a `div` 👍  plus it will be consistent with the rest of the app (btw don't make every change that anyone suggests on your PR, if you disagree or unsure question the reason for the comment, or wait for the maintainers to confirm if the change is required)